### PR TITLE
Сlose time picker on enter

### PR DESCRIPTION
**Pull request by abyr**
Сlose time picker on hit enter

### CODE REVIEW

**Comment by yesmeck** [comment]
Could you add a test?

**Comment by abyr** [comment]
Sure. I've added the test for close on enter. Coverage is also fine now! I hope the change looks a bit better now :)

File: tests/TimePicker.spec.jsx [hunk 1]
```diff
@@ -243,5 +243,26 @@ describe('TimePicker', () => {
         done();
       });
     });
+
+    it('close panel correctly on ENTER', (done) => {
+      const picker = renderPicker();
+      const input = TestUtils.scryRenderedDOMComponentsWithClass(picker,
+        'rc-time-picker-input')[0];
```
Comment (yesmeck) [inline tests/TimePicker.spec.jsx]: Do you mean `rc-time-picker-panel-input`?

Comment (abyr) [reply tests/TimePicker.spec.jsx]: Actually no. But maybe I should?

Comment (yesmeck) [reply tests/TimePicker.spec.jsx]: When panel is open, `rc-time-picker-input` will be covered by the panel, then how can you press <kbd>Enter</kbd> on `rc-time-picker-input`?

Comment (abyr) [reply tests/TimePicker.spec.jsx]: I need to check this a bit later. I thought open/close events are processed in a parent module. Thanks for the note!

Comment (abyr) [reply tests/TimePicker.spec.jsx]: You're right. The first implementation works only on open panel with no focus there through Picker.onKeyDown.
The second test I've just added checks Enter key processing on focused header. The Header is also updated with onEnter prop from Picker.

**Pull request closed without merging**